### DISCUSSION TITLE
code font and code theme

### DISCUSSION THREAD

**Discussion by ShaoClean** [vuepress-theme-hope/vuepress-theme-hope#2738] [Q&A] [answered]
I like font and theme like this:

<img width="766" alt="image" src="https://user-images.githubusercontent.com/98806795/216484309-91d33b66-f8a2-4784-9adb-6cfd4ce5d622.png">

what should I do？

**Comment by Mister-Hope** [comment, marked as answer]
Read the docs. I believe this is not hard for you https://theme-hope.vuejs.press/guide/interface/code-theme.html

**Reply by ShaoClean** [reply]
I want to add custom code theme just like `Monokai`,but it doesn't exist on the theme list

**Reply by Mister-Hope** [reply]
If you would like to create an extra theme, open a pr and I would like yo merge it.
